cmd/otelcol/config/collector/gateway_config.yaml

```diff
@@ -96,6 +96,10 @@ exporters:
     endpoint: "https://ingest.${SPLUNK_REALM}.signalfx.com/v2/trace"
     sending_queue:
       num_consumers: 32
```

Comment:
Let's remove sapm exporter from here. It's not needed anymore

Reply:
ok. Does that apply to the fips/config/gateway_config.yaml